addons/mass_mailing/static/src/js/mass_mailing_html_field.js

```diff
@@ -102,7 +102,7 @@ export class MassMailingHtmlField extends HtmlField {
             const $editable = this.wysiwyg.getEditable();
             this.wysiwyg.odooEditor.historyPauseSteps();
             await this.wysiwyg.cleanForSave();
-            await super.commitChanges(...args);
+            await super.commitChanges({ urgent: true, ...args });
```

Comment:
Are you sure this is going to work? Because ...args is an array, not an object, so the way I see it, spreading it like `{ urgent: true, ...args }` will do something like `{ urgent: true, 0: args[0], 1: args[1], ... }`.

Reply:
@dmo-odoo I updated the args to use the object, but I don't know if it's the cleanest way to do it, so I would appreciate your input here